Source/Particles/Collision/BinaryCollision/BinaryCollisionUtils.H

```diff
@@ -36,6 +38,89 @@ namespace BinaryCollisionUtils{
                                       MultiParticleContainer const * mypc);
 
     CollisionType nuclear_fusion_type_to_collision_type (NuclearFusionType fusion_type);
+
+    /**
+     * \brief Return (relativistic) collision energy, collision speed and
+     * Lorentz factor for transforming between the lab and center-of-momentum
+     * frames. Note the use of `double` for energy since this calculation is
+     * prone to error with single precision.
+     */
+    AMREX_GPU_HOST_DEVICE AMREX_INLINE
+    void get_collision_parameters (
+        const amrex::ParticleReal& u1x, const amrex::ParticleReal& u1y,
+        const amrex::ParticleReal& u1z, const amrex::ParticleReal& u2x,
+        const amrex::ParticleReal& u2y, const amrex::ParticleReal& u2z,
+        const amrex::ParticleReal& m1, const amrex::ParticleReal& m2,
+        amrex::ParticleReal& E_coll, amrex::ParticleReal& v_coll,
+        amrex::ParticleReal& lab_to_COM_factor )
```

Comment:
Consistently with the above remarks, could the variables have more explicit names:
```suggestion
        amrex::ParticleReal& E_kin_COM, amrex::ParticleReal& v_rel_COM,
        amrex::ParticleReal& lab_to_COM_lorentz_factor )
```